pkg/convert/testdata/programs/builtin_functions/main.tf

```diff
@@ -10,14 +10,15 @@ locals {
     # path.root conversion with uses of a path.
     path_root = "root/path"
 
-    # The `can` examples make use of a local `foo`.
-    foo = { "bar" = "baz" }
-
     # The `nonsensitive` examples make use of a local `mixed_content`.
     # We don't use jsondecode(var.mixed_content_json) here because we don't want to depend on the jsondecode function working.
     mixed_content = { "password" = "hunter2" }
 }
 
+variable "can_try_example" {
```

Comment:
This is an auto-generated file, don't edit it manually edit the script that generates it.